rest/blip_sync.go

```diff
@@ -549,7 +550,11 @@ func (bc *blipSyncContext) handleChangesResponse(sender *blip.Sender, response *
 	var answer []interface{}
 	if err := response.ReadJSONBody(&answer); err != nil {
 		body, _ := response.Body()
-		base.ErrorfCtx(bc.blipContextDb.Ctx, "Invalid response to 'changes' message: %s -- %s.  Body: %s", response, err, body)
+		if err == io.EOF {
```

Comment:
I'm not sure how we're ending up with other commits in the PR, especially given it's already been merged.

Let's walk through how you're managing multiple local Git branches in sync up, and see if there's anything we can do to stop this happening.